solutions/bronze/usaco-917.mdx

```diff
@@ -7,7 +7,37 @@ author: Brad Ma, Kevin Sheng, Jay Fu
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_traffic_bronze_feb19.html)
 
+<Warning>
+
+The model solution in the editorial is actually wrong on a couple of test cases.
+For example, for the following input:
+
+```
+2
+on 5 5
+none 3 10
+```
+
+The solution outputs the following:
+
+```
+0 5
+3 10
+```
+
+However, the actual answer is
+
+```
+0 5
+5 10
+```
+
+This is because the starting range in the solution was initialized as ($-\infty$, $\infty$) when in reality it should be (0, $\infty$).
```

Comment:
use `[]` instead of `()` for ranges, since they're inclusive

Reply:
same below

Reply:
should be fixed